examples/json/json.bal

```diff
@@ -1,68 +1,72 @@
 import ballerina/io;
 
 public function main() {
-    // Creates a JSON string value.
+    // Create a JSON string value.
     json j1 = "Apple";
     io:println(j1);
 
-    // Creates a JSON number value.
+    // Create a JSON number value.
     json j2 = 5.36;
     io:println(j2);
 
-    // Creates a JSON true value.
+    // Create a JSON true value.
     json j3 = true;
     io:println(j3);
 
-    // Creates a JSON false value.
+    // Create a JSON false value.
     json j4 = false;
     io:println(j4);
 
-    // Creates a JSON null value.
+    // Create a JSON null value.
     json j5 = null;
 
-    // Creates a JSON Object.
+    // Creates a JSON Object. This is equivalent to a `map<json>`.
     json j6 = { name: "apple", color: "red", price: j2 };
     io:println(j6);
 
-    // Creates a JSON Array. They are arrays including any JSON values.
+    // Create a JSON Array. This is equivalent to a `json[]`.
     json j7 = [1, false, null, "foo", { first: "John", last: "Pala" }];
     io:println(j7);
 
-    // Creates an empty JSON Object.
-    json empty = {};
+    // The `.mergeJson()` method can be used to merge two `json` values.
+    // If either of the two values is `null` the result of `.mergeJson()` is
+    // the other value.
+    json|error j8 = j5.mergeJson(j7);
+    io:println(j8);
 
-    int age = 30;
-    // Creates a JSON object. Keys can be defined with or without quotes.
-    // Values can be any expression.
-    json p = { fname: "John", lname: "Stallone", "age": age };
-    io:println(p);
+    // `.mergeJson()` can also be used to merge two JSON objects.
+    // Where both `json` values are mappings, for each entry in the second,
```

Comment:
```suggestion
    // When both `json` values map, for each entry in the second,
```

Reply:
What we are trying to say is that both of them are mappings of values.